depends/packages/rust.mk

```diff
@@ -1,28 +1,33 @@
 package=rust
-$(package)_version=1.32.0
+$(package)_version=1.42.0
```

Comment:
Is this necessary or optional? It's a big change that will require testing many librustzcash codepaths, that only our lightwalletd server may use

Reply:
The same story, taken from zcash depends. Because it crossbuilds without issues, no other reason.

Reply:
this could break things, we should make upgrading Rust a different PR that can be tested by itself. Many crates depend on Rust version, and even if hushd works perfectly, something in lightwalletd could break. @DenioD you have comments?

Reply:
rust reverted to 1.32.0